code/modules/mining/mine_items.dm

```diff
@@ -104,7 +104,7 @@
 	var/obj/item/stack/flag/F = locate() in get_turf(src)
 
 	var/turf/T = get_turf(src)
-	if(!T || !istype(T,/turf/simulated/floor/asteroid))
+	if(!T || !(istype(T,/turf/simulated/floor/asteroid) || istype(T, /turf/simulated/floor/exoplanet)))
```

Comment:
Id you check istype it automaticaly check isnull

```suggestion
	if(!istype(T,/turf/simulated/floor/asteroid) || istype(T, /turf/simulated/floor/exoplanet))
```

Reply:
Are you sure the ! will apply to (istype() || istype()) and not only to the first istype() ?

Reply:
Ah, forgot it, sorry